tests/deepspeed/test_model_zoo.py

```diff
@@ -97,7 +93,7 @@ def get_launcher(distributed=False):
 
 
 def make_task_cmds():
-    data_dir_fixtures = f"{tests_dir}/fixtures"
+    data_dir_fixtures = "../fixtures"
```

Comment:
I don't think this will work well, you can't use unresolved relative paths in tests since you don't know where the test is running from it could be run from the top-level or from under `tests` or from under `tests/deepspeed/` and `..` will mean totally different things.

Reply:
may be let's leave the paths definitions for now and only change the import and I will clean up this later post your PR merge?